silx/gui/plot/actions/io.py

```diff
@@ -185,10 +188,11 @@ def __init__(self, plot, parent=None):
         self.setShortcut(qt.QKeySequence.Save)
         self.setShortcutContext(qt.Qt.WidgetShortcut)
 
-    def _errorMessage(self, informativeText=''):
+    @staticmethod
+    def _errorMessage(informativeText='', parent=None):
         """Display an error message."""
         # TODO issue with QMessageBox size fixed and too small
-        msg = qt.QMessageBox(self.plot)
```

Comment:
What about:
```
if parent is None:
    parent = self.plot
```
so you keep the previous behavior by default and avoid changing all the calls.